etc/variables.conf

```diff
@@ -7,7 +7,10 @@
 
 # necessary until all affected packages have migrated (bsc#1171164)
 libexec_dirs = /usr/libexec /usr/lib
-# necessary until all affected packages have migrated (bsc#XYZ)
+# necessary until all affected packages have migrated (bsc#1029961)
 sbin_dirs = /sbin /usr/sbin
 # for architecture specific libraries
 lib_dirs = /usr/lib64 /usr/lib
+# for strange cases like mariadb (auth_pam_tool) residing in /usr/lib{,64}
+# and/or /usr/libexec
+lib_libexec_dirs = /usr/lib /usr/lib64 /usr/libexec
```

Comment:
the name is non-intuitive for me. Maybe something like libexec_dirs_64? And can we use $libexec_dirs to define it so that changes there propagate?

Reply:
Yeah, the use case also is non-intuitive :-P.

I doubt we can find a really intuitive name here. If you like yours better then I'm all for it.

Referencing other variables is not supported at the moment and I don't know if it is worth implementing that feature. Hopefully we only maintain a very small list of these variables.

Reply:
your right, lets not waste time on this and just use it as is